sponsor.html

```diff
@@ -0,0 +1,156 @@
+        a wide range of schools and colleges. By partnering with us, you will be
+        able to increase your organisation or product awareness with the next
+        generation of engineers.
+      </div>
+    </div>
+  </div>
+</div>
+<div class="panel panel">
+  <div class="panel-container">
+    <div class="fixed-width">
+      <h2 class="text-center">Sponsor Levels</h2>
+      <div class="row">
+        <div class="column s-4-12 sponsor-level">
+          <h3>Bronze</h3>
+          <h4>£500 or more</h4>
+          <ul>
+            <li>Link and logo on our website</li>
+            <li>Company name, logo and brief overview in our competition booklet</li>
+            <li>Invite for a representative to attend the competition</li>
+          </ul>
+        </div>
+        <div class="column s-4-12 sponsor-level">
+          <h3>Silver</h3>
+          <h4>£1,000 or more</h4>
+          <ul>
+            <li>All the perks of Bronze sponsorship</li>
+            <li>Friendly mention on social media</li>
+            <li>Logo on a banner at the competition</li>
+            <li>An opportunity during the year to present your organisation to our student competitors</li>
+          </ul>
+        </div>
+        <div class="column s-4-12 sponsor-level">
+          <h3>Gold</h3>
+          <h4>£2,000 or more</h4>
+          <ul>
+            <li>All the perks of Silver sponsorship</li>
+            <li>Logo on the volunteer t-shirts</li>
+            <li>Friendly mention and thank you at the competition</li>
+            <li>Sponsor an award - put your name to one of our existing awards, or suggest a new award and define the judging criteria*</li>
+          </ul>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column d-12-12 sponsor-level">
+          <h3>Platinum</h3>
+          <h4>£10,000 or more</h4>
+            <ul>
+              <li>All the perks of Gold sponsorship</li>
+              <li>Dedicated space at the competition</li>
+              <li>Set a challenge for teams, and declare the winner at the competition**</li>
+              <li>Short (~5 min.) pre-recorded presentation at the start of the competition, to talk about your company and set your challenge, and a short in-person presentation on the final weekend to announce the winner</li>
+            </ul>
+          <p style="margin-top: 1rem;">
+            If there is anything not mentioned we can offer you or you would be interested in?? Get in touch!
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="panel">
+  <div class="panel-container">
+    <div class="fixed-width text-center">
+      <a name="signup"></a><h2>Want to sponsor us?</h2>
+      <div class="column m-8-12 m-offset-2-12 text-left">
+        <p>
+          Please fill in the form below to and we will get in touch with you soon!
+        </p>
+        <!--TODO: Create sponsor sign up form-->
+        {% include team-signup.html %}
```

Comment:
Rather than a form, should we just push people to `fundraising@`? For sums of money like that, I feel like they deserve some kind of human interaction

Reply:
I agree that having a form feels like it might be a bit impersonal especialy with the mailchimp notice below it (feels more like signing up for a maiing list than starting a meaningfull discussion), If the form is still wanted i have a few suggestions below.

* Add somehting like the following infront of it. "If you are interested in sponsoring us or want more information on sponsoring drop us an email at fundraising@studentrobotics.org or fill in the form below". That way there is a less constrained way for them to contact us. 
* Consider merging the first and last name boxes into just name as having them seaperated makes is seem like a form for sigining up for somthing.
* Add a free text box to allow the to start the conversation if they wantrather than waiting for us to get back to them to start it.

Reply:
It depends entirely on the amount of money, and how significant the amount is to the sponsor. They might be giving £500 and have £1M to give to charities, at which point they may just want to click the relevant buttons and be done with it. Requiring more than necessary human interaction may put them off sponsoring. As such having both e-mail and form available gives the flexibility.

I agree the structure of it could change, as well as rewording the MailChimp notice.